### PR TITLE
Fix to PR #616

### PR DESCRIPTION
**Pull request by tomasznapieralski**
Changes summary:
- removed fail check <-- [QAART-576](https://github.com/Wikia/selenium-tests/tree/QAART-576) fix that problem (already included);
- fixed unhandled exception from loading spinner;
- added PageObjectLogging::logWarning() method, which adds yellow line to log;
- added PageObjectLogging::log() method, which will log different message on success and fail to log;
- added BasePageObject::waitForElementVisibleByElement() method, which will wait declared time;
- replaced logging with if..else to new log() method;

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
**DON'T MERGE BEFORE QAART-576**

**Comment by rochan** [comment]
@tomasznapieralski  @ludwikkazmierczak  Is this https://github.com/Wikia/selenium-tests/commit/8072ec3d8394339f5008a2c9c7a17663cf8729cc an intentional commit?

File: src/test/java/com/wikia/webdriver/common/core/exceptions/TestFailed.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.wikia.webdriver.common.core.exceptions;
+
+/**
+ * Created by Ludwik on 2015-04-14.
+ */
+public class TestFailed extends Throwable {
```
Comment (rochan) [inline src/test/java/com/wikia/webdriver/common/core/exceptions/TestFailed.java]: Question: Is there something specific of this class that it needs to extend Throwable and not Exception?

Comment (rochan) [inline src/test/java/com/wikia/webdriver/common/core/exceptions/TestFailed.java]: Class name of this should reflect that it is a throwable or exception

**Comment by tomasznapieralski** [comment]
Fix is here #621

**Pull request closed without merging**